### PR TITLE
bonjour: add a debug print when connection refused

### PR DESCRIPTION
**Pull request by yotamolenik**

### CODE REVIEW

File: pymobiledevice3/bonjour.py [hunk 1]
```diff
@@ -59,6 +59,7 @@ def add_service(self, zc: Zeroconf, type_: str, name: str) -> None:
                 if lockdown.paired:
                     break
         except ConnectionRefusedError:
+            logger.debug('connection refused')
```
Comment (doronz88) [inline pymobiledevice3/bonjour.py]: - add context
- `failed to establish a lockdown connection`

**Review by doronz88 — CHANGES_REQUESTED** [review]

File: pymobiledevice3/bonjour.py (hunk 1 above)
Comment (yotamolenik) [reply pymobiledevice3/bonjour.py]: done

**Review by doronz88 — APPROVED** [review]

File: pymobiledevice3/bonjour.py [hunk 2]
```diff
@@ -59,6 +59,7 @@ def add_service(self, zc: Zeroconf, type_: str, name: str) -> None:
                 if lockdown.paired:
                     break
         except ConnectionRefusedError:
+            logger.debug('failed to establish a lockdown connection')
```
Comment (matan1008) [inline pymobiledevice3/bonjour.py]: 'Service failed to establish a lockdown connection'

Comment (yotamolenik) [reply pymobiledevice3/bonjour.py]: done

**Review by matan1008 — APPROVED** [review]

**Pull request merged**